dapp/src/utils/cardano/constants.ts

```diff
@@ -0,0 +1,33 @@
+import { CardanoWalletInfo } from "./types";
+
+export const cardanoApiMinSlot = 40000000;

```

Comment:
Where does this come from?

Reply:
This is used in the request to Carp for projected nft data, it is a required parameter:
```
const request = await axios.post(
      `${env.REACT_APP_CARDANO_API_URL_BASE}/projected-nft/range`,
      { range: { minSlot: cardanoApiMinSlot, maxSlot: cardanoApiMaxSlot } },
    );
```
First projected nft data on preprod happened at slot 43201687. It should ideally be set to the slot from which we start to have some projected nft data. But I don't know if it makes any performance difference if there are no data before the set minSlot anyway, or if it is somehow wasteful to set it to 1. Maybe @gostkin can answer this?

Reply:
performance-wise it doesn't matter if it is 1 or 40000000. one thing to keep in mind:

>  make sure you don't try to get all projected nft data in a single request.

Reply:
the more activity comes to the projected nft the more data needs to be returned

Reply:
> make sure you don't try to get all projected nft data in a single request.  

@gostkin Can you elaborate on this please? Are/Will there be some limits? Or any pagination system? What range should I use then?
In that case it would be good if the Carp endpoint supported filtering for a given ownerAddress via request parameter, since the dApp needs data for the connected user only.

Reply:
If any pagination system is required, I would rather we implement the pagination like the Carp client logic (like we did for pagination of other endpoints) since it shouldn't be the responsibility of the dApp to know how to paginate the requests properly (the Carp client should know how to paginate its own endpoint)

Reply:
@matejos you can do 
```sh
curl -X POST -H 'Content-Type: application/json' -d '{"range":{"minSlot":4606812,"maxSlot":46706812},"address":"9040f057461d9adc09108fe5cb630077cf75c6e981d3ed91f6fb18f6"}' 0.0.0.0:3000/projected-nft/range
```

to query by specific address, this is rolled out on dev carp, so you can test